modules/ROOT/nav.adoc

```diff
@@ -10,6 +10,10 @@
 ** xref:quickstart:local-quickstart.adoc[Local Development]
 ** xref:quickstart:network-quickstart.adoc[Network Deployment]
 
+* xref:examples:index.adoc[Example Code]
+*** link:https://github.com/dfinity/examples[Dfinity Examples^]
+*** link:https://github.com/dfinity/awesome-dfinity[Open-source Community Projects^]
```

Comment:
This repository hasn't been updated in 4 months now. We could try and maintain a list of good quality _and recent_ open source projects ourselves.